src/core/thread/network_data_leader_ftd.cpp

```diff
@@ -270,7 +280,9 @@ void Leader::HandleCommissioningSet(Coap::Header &aHeader, Message &aMessage, co
         }
     }
 
-    SetCommissioningData(tlvs, length);
+    SetCommissioningData(tlvs, static_cast<uint8_t>(length));
```

Comment:
shall we verify `length` is not bigger than 255 because it's increased at [line 277](https://github.com/jwhui/openthread/blob/fe1e6b39b584334767e4e1b8382d4a639b88aec8/src/core/thread/network_data_leader_ftd.cpp#L277).